core/arch/arm/kernel/stmm_sp.c

```diff
@@ -862,7 +867,7 @@ static bool spm_handle_svc(struct thread_svc_regs *regs)
 	}
 }
 
-const struct ts_ops stmm_sp_ops __rodata_unpaged = {
```

Comment:
I believe we need to keep the `__rodata_unpaged` as `dump_state` is accessed from `abort_print_current_ts()`

Reply:
Right, I missed that!

Reply:
Removing `const` would do the job. Would that be ok to you (and others)?

Reply:
Why would we want to do that?

Reply:
to move it to a read/write unpaged section of the pager.

Reply:
I got that part. But why would we want to have it read/write when it's more secure to have it read-only?

Reply:
maybe only upon `CFG_WITH_PAGER=y`, as a `DECLARE_KEEP_PAGER()` for the dump function handler.

Reply:
`__rodata_unpaged` only makes this struct unpaged, not it's dependencies. Keeping `__rodata_unpaged` and adding `DECLARE_KEEP_PAGER()` for `stmm_dump_state()` should do the trick.

Reply:
fixup commit added.